web/src/components/PublicationsChapterMenu/PublicationsChapterMenu.tsx

```diff
@@ -15,23 +15,27 @@ export const PublicationsChapterMenu: FC<PublicationsChapterMenuProps> = ({
 	ariaLabel,
 	chapters,
 }) => {
-	const router = useRouter();
+	const { asPath } = useRouter();
 
 	return (
-		<StackedNav aria-label={ariaLabel}>
-			{chapters.map((item) => {
-				const destination = item.url;
-				return (
-					<StackedNavLink
-						key={item.url}
-						destination={destination}
-						elementType={ScrollToContentStartLink}
-						isCurrent={destination === router.asPath}
-					>
-						<span dangerouslySetInnerHTML={{ __html: item.title }} />
-					</StackedNavLink>
-				);
-			})}
-		</StackedNav>
+		<>
```

Comment:
Can we remove the extra fragment please? It will hopefully have the benefit of the diff being easier to view!